### PR TITLE
PROGRAMMES-5749 - Recipes

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: app/Resources/views/find_by_pid/tlec.html.twig [hunk 1]
```diff
@@ -32,26 +32,42 @@
...
-                </div>
+                </ul>
             {%- endif -%}
 
             {%- if promotions or clips or galleries -%}
                 <ul class="grid">
+                    {%- if programme.getOption('recipes_enabled') -%}
+                        <li class="grid__item 1/2@gel3 1/3@gel3b 1/4@gel5 gel-brevier-bold lazy-module"
+                             data-lazyload-inc="{{ path('partial_programme_recipes', {'pid': programme.getPid()}) }}"
+                             data-lazyload-threshold="768"
```
Comment (BjornTwachtmann) [inline app/Resources/views/find_by_pid/tlec.html.twig]: Can you reduce this to 320 in line with clips and galleries (in V2), I know that's not what it is currently. That is my stupid fault. But it should be 320 I think.

File: app/Resources/views/find_by_pid/tlec.html.twig [hunk 2]
```diff
@@ -32,26 +32,42 @@
...
+                             data-lazyload-delay="true">
+                            <div class="media media--column media--card br-box-secondary">
+                                <a class="media__body link-complex" href="{{ path('programme_recipes', {'pid': programme.getPid()}) }}">
+                                    <div class="media__details">
+                                        <div class="media__meta-group link-complex__target">
+                                            <h4 class="gel-pica-bold br-secondary-text-ontext">{{ tr('recipes') }}</h4>
+                                            <p>{{ tr('see_all_recipes_from', {'%1': programme.getTitle()}) }}</p>
```
Comment (BjornTwachtmann) [inline app/Resources/views/find_by_pid/tlec.html.twig]: Formatting here with JS off is slightly screwed. E.g. white recipes title has coloured underline. Trailing link text is too small and weirdly spaced.

File: app/Resources/views/partial/on_now_legacy_amen.html.twig [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{% extends 'base_ds2013_partial.html.twig' %}
```
Comment (BjornTwachtmann) [inline app/Resources/views/partial/on_now_legacy_amen.html.twig]: Shouldn't this use the amen one?

**Comment by BjornTwachtmann** [comment]
Other than the stuff listed this looks good

**Pull request merged**